### DISCUSSION TITLE
Getting SCIP log output

### DISCUSSION THREAD

**Discussion by ahmtsn** [google/or-tools#3412] [Linear Solver questions] [answered]
Hello.
I'm using Google OR-tools in Java for modeling an MIP and selected SCIP as the solver. I created solver and attempted to pass parameters to SCIP as follows:

`MPSolver solver = MPSolver.createSolver("SCIP");
...
solver.setSolverSpesificParametersAsString("display/verblevel=5");`

When I run the model, it runs without any warning or error and i can obtain info about the solution, but I am not able to obtain any output in terminal about solve and search process (presolve, b&b, nodes, gap, lb, un etc.). In Java how can I generate SCIP output?
Sincerely

**Comment by lperron** [comment, marked as answer]
solver.EnableOutput()

**Reply by ahmtsn** [reply]
Thank you. It was there but I couldn't find it somehow. When there are more than one thread is `solver.enableOutput()` enough, or do I have to set parameters to obtain output from concurrent solvers?
